markdownlint/TOP009_lessonOverviewItemsSentenceStructure/tests/TOP009_capital_letter.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+
+This file should be marked with an error because one of the lesson overview items is not properly capitalised.
```

Comment:
```suggestion
This file should be marked with an error because one of the lesson overview items is not properly capitalized.
```
Unfortunately, we Brits must conform